device/transport/mqtt/test/_mqtt_test.js

```diff
@@ -637,9 +637,9 @@ describe('Mqtt', function () {
     });
 
     /* Tests_SRS_NODE_DEVICE_MQTT_41_XXX: [For a Plug and Play Device the device capability model should be included as &digital-twin-model-id=<DEVICE’s DCM ID> after the api-version]*/
```

Comment:
>evice capability model [](start = 75, length = 22)

Change the comment.  Don't think this spec comment was actually in the .ts file.